lib/paypal_nvp.rb

```diff
@@ -23,16 +24,22 @@ def initialize(sandbox = false, extras = {})
       @url  = config[type]["url"]
       @user = config[type]["user"]
       @pass = config[type]["pass"]
-      @cert = config[type]["cert"]
+      @signature = config[type]["signature"]
+      @key_path = config[type]["key_path"]
+      @cert_path = config[type]["cert_path"]
       @rootCA = config[type]["rootca"]
       @open_timeout = config[type]["open_timeout"]
       @read_timeout = config[type]["read_timeout"]
     else
-      @url  = extras[:url]
-      @user = extras[:user]
-      @pass = extras[:pass]
-      @cert = extras[:cert]
-      @rootCA = extras[:rootca]
+      @url  = extras.delete(:url)
+      @user = extras.delete(:user)
```

Comment:
Use delete so irrelevant params [won't be included in the request params](https://github.com/fac/paypal_nvp/pull/3/files#diff-ff70e0ad158bb2e97f36ffe1a2e42ebaL51)